### PR TITLE
[HEVCe] Add Doc Description for Featues with LowPower ON

### PR DESCRIPTION
**Pull request by JeremyShang**
Features Including: (1) LCUSize (2) Target encoding chroma format
(3) QP range (4) Active reference number (5) Target encoding bitdepth

### CODE REVIEW

**Review by onabiull — APPROVED** [review]

File: doc/mediasdk-man.md [hunk 1]
```diff
@@ -4148,7 +4148,7 @@ The application can attach this extended buffer to the [mfxVideoParam](#mfxVideo
 `MaxFrameSizeP` | Same as [mfxExtCodingOption2](#mfxExtCodingOption2)`::MaxFrameSize` but affects only P/B-frames. If MaxFrameSizeP equals 0, the SDK sets MaxFrameSizeP equal to MaxFrameSizeI. If MaxFrameSizeP is not specified or greater than spec limitation, spec limitation will be applied to the sizes of P/B-frames.
 `EnableQPOffset` | Enables `QPOffset` control.<br><br>See the [CodingOptionValue](#CodingOptionValue) enumerator for values of this option.
 `QPOffset` | When `EnableQPOffset` set to ON and RateControlMethod is CQP specifies QP offset per pyramid layer.<br>For B-pyramid, B-frame QP = QPB + QPOffset[layer].<br>For P-pyramid, P-frame QP = QPP + QPOffset[layer].
-`NumRefActiveP`, `NumRefActiveBL0`, `NumRefActiveBL1` | Max number of active references for P and B frames in reference picture lists 0 and 1 correspondingly. Array index is pyramid layer.
+`NumRefActiveP`, `NumRefActiveBL0`, `NumRefActiveBL1` | Max number of active references for P and B frames in reference picture lists 0 and 1 correspondingly. Array index is pyramid layer. Use Query function to check the maximum values.
```
Comment (MaximIntel) [inline doc/mediasdk-man.md:4151]: `Use [Query](#MFXVideoENCODE_Query) function to check ...`

**Review by MaximIntel — CHANGES_REQUESTED** [review]

File: doc/mediasdk-man.md (hunk 1 above)
Comment (JeremyShang) [reply doc/mediasdk-man.md]: Thanks, fixed

**Review by vilichev — APPROVED** [review]

**Review by MaximIntel — DISMISSED** [review]

File: doc/mediasdk-man.md [hunk 2]
```diff
@@ -4161,9 +4161,9 @@ The application can attach this extended buffer to the [mfxVideoParam](#mfxVideo
...
-`TargetBitDepthLuma` | Target encoding bit-depth for luma samples. May differ from source one. `0` mean default target bit-depth which is equal to source ([mfxVideoParam::](#mfxVideoParam)[mfx::](#mfxInfoMFX)[FrameInfo::](#mfxFrameInfo)`BitDepthLuma`).
-`TargetBitDepthChroma` | Target encoding bit-depth for chroma samples. May differ from source one. `0` mean default target bit-depth which is equal to source ([mfxVideoParam::](#mfxVideoParam)[mfx::](#mfxInfoMFX)[FrameInfo::](#mfxFrameInfo)`BitDepthChroma`).
+`TargetChromaFormatPlus1` | Minus 1 specifies target encoding chroma format (see [ChromaFormatIdc](#ChromaFormatIdc) enumerator). May differ from source one. `TargetChromaFormatPlus1 = 0` mean default target chroma format which is equal to source ([mfxVideoParam::](#mfxVideoParam)[mfx::](#mfxInfoMFX)[FrameInfo::](#mfxFrameInfo)`ChromaFormat + 1`), except RGB4 source format.<br>In case of RGB4 source format default target chroma format is 4:2:0 (instead of 4:4:4) for the purpose of backward compatibility. For HEVC encoder with LowPower ON, MFX_CHROMAFORMAT_YUV422 is not supported.
```
Comment (thomasli21801) [inline doc/mediasdk-man.md:4164]: probably query is a way to avoid change again after 422 enabling?

**Review by thomasli21801 — DISMISSED** [review]

**Review by cchunbo — DISMISSED** [review]

**Review by dmitryermilov — CHANGES_REQUESTED** [review]
Hi @JeremyShang 
I fully support and appreciate describing encoder's limitation into public documentation however I'm not sure that the manual is the right place.  Let's double check together. The main reason is we attempt to keep the manual platforms agnostic. Plus, ideally we need to same changes for VPL.
Can you please check with @eistarov  and @varistar if this change should land into the spec or better to create some HEVC specific release note document? 
Blocking the change until we get comments from Vasily and Evgeniy

**Comment by JeremyShang** [comment]
Hi @dmitryermilov , I get it. Sure, I will check with them. Thanks!

**Comment by JeremyShang** [comment]
Hi @dmitryermilov, @varistar , I've moved encoder's limitation description to the mediasdk_release_notes.md(HEVC encoder part). May I ask is it the right place?

**Comment by JeremyShang** [comment]
Hi @eistarov , @varistar , @dmitryermilov , may I ask your help to take a look at the latest commit 91a7a28  ?

**Review by varistar — APPROVED** [review]

**Review by thomasli21801 — APPROVED** [review]

**Review by dmitryermilov — APPROVED** [review]

**Comment by samuel100u** [comment]
I set LUC = 32x32  when encode hevc with low power on. And GPU hang.....

So LCU need to be 64x64 with low power on, right?

**Comment by thomasli21801** [comment]
> I set LUC = 32x32 when encode hevc with low power on. And GPU hang.....
> 
> So LCU need to be 64x64 with low power on, right?

Hi Samuel,
You are right. Low Power On supports LCU 64x64 only.

**Comment by JeremyShang** [comment]
> I set LUC = 32x32 when encode hevc with low power on. And GPU hang.....
> 
> So LCU need to be 64x64 with low power on, right?

Hi Samuel, Yes, LCU need to be 64x64 with low power on. May I ask on which HW platform do you observe such GPU hang (Ice Lake, Tiger Lake, or others) and which app you used(sample_encode? ) I will try to reproduce that hang. Thanks!

**Comment by samuel100u** [comment]
> > I set LUC = 32x32 when encode hevc with low power on. And GPU hang.....
> > 
> > So LCU need to be 64x64 with low power on, right?
> 
> Hi Samuel, Yes, LCU need to be 64x64 with low power on. May I ask on which HW platform do you observe such GPU hang (Ice Lake, Tiger Lake, or others) and which app you used(sample_encode? ) I will try to reproduce that hang. Thanks!

DG1

**Pull request merged**